modules/kmm-using-signing-with-kmm.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * hardware_enablement/kmm-kernel-module-management.adoc
+
+:_content-type: CONCEPT
+[id="kmm-using-signing-with-kmm_{context}"]
+= Using signing with Kernel Module Management (KMM)
+
+On a Secure Boot enabled system, all kernel modules (kmods) must be signed with a public/private key-pair
+enrolled into the Machine Owner's Key (MOK) database. Drivers distributed as part of a distribution should already be signed by the distribution's private key,
+but for kernel modules build out-of-tree, KMM supports signing kernel modules using the `sign` section of the kernel mapping.
+
+For more details on using Secure Boot, see link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/managing_monitoring_and_updating_the_kernel/signing-a-kernel-and-modules-for-secure-boot_managing-monitoring-and-updating-the-kernel#generating-a-public-and-private-key-pair_signing-a-kernel-and-modules-for-secure-boot[Generating a public and private key pair]
+
+.Prerequisites
+
+To use this functionality you need:
```

Comment:
IBM SG says avoid continuing a lead-in sentence into a list. So something like: "To use this functionality, you must have the following:"

But since this is already clearly a `.Prerequisites` list, you could probably just drop the sentence?

Reply:
Dropped the sentence.